src/plugins/saved_objects_management/public/management_section/objects_table/components/relationships.tsx

```diff
@@ -116,7 +116,7 @@ export class Relationships extends Component<RelationshipsProps, RelationshipsSt
         }
         color="danger"
       >
-        {error}
+        <p>{error}</p>
```

Comment:
`{error}` should be wrapped in `<p></p>`, in order to indicate a paragraph of text